client/object_delete.go

```diff
@@ -72,6 +72,12 @@ func (x *PrmObjectDelete) ByID(id oid.ID) {
 	x.addr.SetObjectID(&idV2)
 }
 
+// ByAddress specifies address of the requested object.
+// Required parameter. It is an alternative to ByID, FromContainer functions.
```

Comment:
I think we need some cross-referencing from ByID and FromContainer as well (mention in comments, otherwise we have all of them listed as required). And btw, will these be linked properly in godoc (I'm not entirely sure what's the correct syntax for methods)?

Reply:
About godoc, looks like there is no way "to link" function to another one. I've checked standard lib and any function reference represented like the plain text:
* https://pkg.go.dev/strings#SplitN (Edge cases for s and sep (for example, empty strings) are handled as described in the documentation for Split.)

Split - is the another function from the same package in we have highlights in IDE, but no inside godoc

Reply:
Wonderful new world! Updated docs to `links`. Looks like this trick doesn't work with unexported structs. `PrmObjectGet`, `PrmObjectHead`, `PrmObjectRange` have embedded `prmObjectRead` which contains our functions `FromContainer, ByID, ByAddress` with documentation. Referencing by `[prmObjectRead.ByAddress]` makes zero effect in godoc for public types. All functions is plain text

Reply:
Private ones are private, it's not critical, but public ones should be documented properly.